akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/SerializationSupport.scala

```diff
@@ -56,8 +56,8 @@ trait SerializationSupport {
   def compress(msg: MessageLite): Array[Byte] = {
     val bos = new ByteArrayOutputStream(BufferSize)
     val zip = new GZIPOutputStream(bos)
-    msg.writeTo(zip)
-    zip.close()
+    try msg.writeTo(zip)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/vasseurjb/akka/file/2046293339/issues/source?bid=3132306&fileBranchId=3132365#l59)